### ISSUE TITLE
AR Model Bug Extreme Overfitting

### ISSUE THREAD

**Issue by shamalmhetre** [statsmodels/statsmodels#5446]
Extreme out of sample prediction

####################################################################

from statsmodels.tsa.ar_model import AR  #### Statsmodel 0.9.0
import matplotlib.pyplot as plt

data=[2.328749,
2.853576,
3.246347,
5.027859,
3.497954,
3.97903,
3.675557,
3.438269,
4.925348,
3.394905,
3.733191,
4.168234,
3.772922,
4.260573,
5.395185,
3.587519,
4.587907]

xdata=[x  for x in range(1, 18)]
print(xdata)
print(data)

plt.plot(xdata, data, marker='x', label='linear')
plt.plot()
plt.show()

# fit model
model = AR(data)
model_fit = model.fit()
# make prediction
yhat = model_fit.predict(len(data), len(data))
print("Prediction")
print(yhat)

Prediction
[26.99972292]

####################################################################

############### Feeding first 12 values

from statsmodels.tsa.ar_model import AR  #### Statsmodel 0.9.0
import matplotlib.pyplot as plt

data=[2.328749,
2.853576,
3.246347,
5.027859,
3.497954,
3.97903,
3.675557,
3.438269,
4.925348,
3.394905,
3.733191,
4.168234]

xdata=[x  for x in range(1, 13)]
print(xdata)
print(data)

plt.plot(xdata, data, marker='x', label='linear')
plt.plot()
plt.show()

# fit model
model = AR(data)
model_fit = model.fit()
# make prediction
yhat = model_fit.predict(len(data), len(data))
print("Prediction")
print(yhat)

Prediction
[3.2760281]

##########################################

**Comment by Tom-McDonald** [comment]
I would suggest this is likely because the data is non-stationary, doing a quick `adfuller()` stationarity test returns:
```
ADF Statistic: 0.000000
p-value: 0.958532
Critical Values:
	1%: -4.665
	5%: -3.367
	10%: -2.803
```

So maybe try differencing the data first before fitting a model.

**Comment by bashtage** [comment]
Closing as answered.
